### ISSUE TITLE
useFullscreen: Module not found: Can't resolve 'tslib'

### ISSUE THREAD

**Issue by gpoole** [imbhargav5/rooks#449]
**Describe the bug**
Installing @rooks/use-fullscreen and importing into my Next.js app causes the following compile error:

```
error - ./node_modules/@rooks/use-fullscreen/lib/index.esm.js:1:0
Module not found: Can't resolve 'tslib'
```

I can see that `index.esm.js` is importing `tslib`:

```
import { __awaiter } from 'tslib';
```

But `tslib` is not a dependency of @rooks/use-fullscreen.

Installing `tslib` manually in my project fixes the problem.

**Comment by imbhargav5** [comment]
Interesting. Thanks! @gpoole  Will take a look.

**Comment by imbhargav5** [comment]
@gpoole Do you see this issue with `v5`? In v5 we are just using a single package `rooks` and we solved a few issues around types etc. Please check if this issue was resolved. Else we will attempt a new PR fix.

**Comment by gpoole** [comment]
Yes this is fixed in v5, thanks @imbhargav5. I should have specified in the original report but the problem I had was with 4.11.0 but still a problem in 4.11.2.

I think it could be helpful for existing users to npm-deprecate the v4 separate packages? I only realised v5 was out when you asked me to check it and I couldn't install it with `npm i @rooks/use-fullscreen@latest` so I had a look on the npmjs.com page.

**Comment by gpoole** [comment]
That makes sense, thanks @imbhargav5
